### ISSUE TITLE
Issue with Secret of Monkey Island

### ISSUE THREAD

**Issue by Eddepad** [randyrossi/bmc64#258]
Copy from my post on Lemon, just curious what this issue can be:

Can anyone reproduce this one?

With the game Secret of Monkey Island, the skull pointer in the game cannot be moved in any direction (?)
Game for free download at [https://csdb.dk/release/?id=237672](url)

Don't know if it's related to BMC64 or maybe a setting. I play with PAL and USB speedlink joystick. I don't have issues with all other games, so this one is weird...

**Comment by nickgoodmanuk** [comment]
Have you tried it on any other VICE based emulators, like winvice ?

**Comment by Eddepad** [comment]
> Have you tried it on any other VICE based emulators, like winvice ?

Yes, with (one of the) latest Vice and Denise. No problems there.

No issues with your BMC I presume?

**Comment by Eddepad** [comment]
@randyrossi 
Thanks for your reply. I just tried WinVice 3.3 on a PC the x64.exe, with NumPad as joystick, and the game works (the mouse/skull pointer can move). 

Tested at 3.4 on the x64sc.exe and that worked also. The x64.exe was not available in the download.
